config/vufind/facets.ini

```diff
@@ -230,11 +230,20 @@ special_facets   = "illustrated,daterange"
 ; language files found in the languages directory. You may add a colon and the
 ; name of a text domain after the field name to specify translations in a specific
 ; text domain (subdirectory of the languages folder). If you comment out the lines
-; below, no facets will be translated.
+; below, no facets will be translated. If you add a second colon, you may add a
+; format descriptor containing the placeholders %%raw%% and %%translated%%. This can
```

Comment:
I've tested that it now works to either directly place the format descriptor here, or to use a translation string. We should probably adjust this documentation accordingly. Maybe something like:

If you add a second colon, you may add a translation string containing the placeholders %%raw%% and %%translated%%. You can enter the translation string directly (if the formatting is uniform for all languages), or you can use a key pointing to values found in the language files (if you need different formatting for different languages).

(...and then the text can continue as it presently exists).

What do you think?

Reply:
Sounds good to me!

Reply:
Thanks! I've incorporated the proposed text and done a bit of reformatting to improve the readability of the documentation comment.